scripts/opt-alive.sh.in

```diff
@@ -14,6 +14,15 @@ PASSES="-passes= argpromotion deadargelim globalopt hotcoldsplit inline ipconstp
 
 TV="-tv"
 for arg in $@; do
+  # update the -passes argument only.
+  shift
+  if [[ $arg == "-passes="* ]]; then
+    passes=${arg#-passes=}
+    passes_new=`@CMAKE_BINARY_DIR@/../scripts/rewritepass.py @LLVM_BINARY_DIR@/bin/opt $passes`
+    set -- "$@" "-passes=module(tv),$passes_new,module(tv)"
```

Comment:
Even if `tv` is a module-level pass, omitting this `module` adaptor made tests like `opt-tv/npm.opt.ll` fail.
I think there is something with this `module` keyword.

Reply:
Can we make the addition of tv conditional on whether $passes is a module pass or not? There's no point in calling alive for globalopt & friends.
This could be done in python.